### PR TITLE
Add brocado, cabrera, maroca, migué and more...

### PR DESCRIPTION
**Pull request by welksonramos**

### CODE REVIEW

File: words.json [hunk 1]
```diff
@@ -60,12 +74,16 @@
   "lantejoula",
   "limitrofe",
   "malaco",
+  "maroca",
   "meleca",
   "menisquencia",
-  "mentecapto",
+  "mentecapto"
```
Comment (Ghostavio) [inline words.json]: opa, faltou uma vírgula aqui, não?

Comment (welksonramos) [inline words.json]: verdade, falha minha...

**Comment by rafaelrinaldi** [comment]
@welksonramos Thanks for contributing! The CI is breaking. Can you please review? If you have some questions the [contributing guide](https://github.com/rafaelrinaldi/porreta/blob/master/CONTRIBUTING.md) might help you out :+1:

**Comment by welksonramos** [comment]
nice! +1:

**Comment by rafaelrinaldi** [comment]
@welksonramos Hey man. There are still conflicts on this pull request. Can you take a look?

**Comment by rafaelrinaldi** [comment]
@welksonramos You are more than welcome to resend this PR. I'm closing it because it's not in sync with `master` and has a few issues  with the CI.

**Pull request closed without merging**